http-clients/src/test/java/com/palantir/remoting3/clients/UserAgentTest.java

```diff
@@ -24,21 +24,21 @@
 public final class UserAgentTest {
 
     @Test
-    public void testCorrectHeaderFormatWithInstanceId() {
-        UserAgent baseUserAgent = UserAgent.of(UserAgent.Agent.of("service", "1.0.0"), "instanceId");
-        assertThat(UserAgents.format(baseUserAgent)).isEqualTo("service/instanceId (1.0.0)");
+    public void testCorrectHeaderFormatWithNodeId() {
+        UserAgent baseUserAgent = UserAgent.of(UserAgent.Agent.of("service", "1.0.0"), "nodeId");
```

Comment:
can we use a nodeId string that's not `nodeId` so that we know this actually gets rendered?

Reply:
sure